src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -87,19 +87,18 @@ class HoldRequest extends React.Component {
    */
   getNotification() {
     // Show holiday schedule warning if current date strictly less than
-    // 2018-11-27 (i.e. don't show notification after 2018-11-26 23:59:59)
-    const isNov27OrLater = new Date() >= new Date(2018, 10, 27)
-    if (isNov27OrLater) {
+    // 2019-01-02 (i.e. don't show notification after 2019-01-01 23:59:59)
+    const is2019Jan2OrLater = new Date() >= new Date(2019, 0, 2)
```

Comment:
Javascript really makes January month number 0? I think some people take the starting at 0 thing a little too far